charts/authentik/values.yaml

```diff
@@ -155,6 +155,10 @@ authentik:
   # -- Secret key used for cookie singing and unique user IDs,
   # don't change this after the first install
   secret_key: ""
+  ## use existing secret for authentik configuration instead of creating one
```

Comment:
Let's add a warning that it renders whatever is set under the `authentik` key unused.